packages/devextreme-vue/eslint.config.mjs

```diff
@@ -0,0 +1,220 @@
+            "curly": ["error", "multi-line", "consistent"],
+            "unicode-bom": ["error", "never"],
+            "eol-last": ["error", "always"],
+            "indent": ["error", 4, {
+                SwitchCase: 1,
+                MemberExpression: 1,
+                CallExpression: { arguments: 1 },
+            }],
+            "quotes": ["error", "single"],
+            "import/named": "error",
+            "import/default": "error",
+            "import/no-duplicates": "error",
+        },
+    },
+    ...compat.extends('devextreme/typescript').map(config => {
+      const newConfig = {
+        ...config,
+        files: ['**/*.ts?(x)']
+      };
+
+      if (config.rules) {
+        newConfig.rules = processDevExtremeRules(config.rules);
+      }
+
+      return newConfig;
+    }),
+    {
+        files: ['**/*.ts?(x)'],
+        languageOptions: {
+            parser: tsParser,
+            parserOptions: {
+                createDefaultProgram: true,
+                project: "./tsconfig.json",
+                tsconfigRootDir: __dirname,
+                ecmaVersion: 6,
+                sourceType: "module",
+            },
+        },
+        plugins: { 
+          "@typescript-eslint": ts,
+        },
+        rules: {
+            "prefer-regex-literals": "off",
+            "i18n/no-russian-character": ["error", { includeIdentifier: true }],
+            "max-len": ["error", { code: 150 }],
+            "@typescript-eslint/no-explicit-any": "warn",
+            "spellcheck/spell-checker": "warn",
+            "@typescript-eslint/prefer-optional-chain": "warn",
+            "@typescript-eslint/no-unsafe-return": "warn",
+            "@typescript-eslint/explicit-function-return-type": "warn",
+            "@typescript-eslint/no-use-before-define": "warn",
+            "no-param-reassign": "warn",
+            "consistent-return": "warn",
+            "array-callback-return": "warn",
+            "no-underscore-dangle": "warn",
+            "no-prototype-builtins": "warn",
+            "@typescript-eslint/explicit-module-boundary-types": "warn",
+            "no-restricted-syntax": "warn",
+            "import/no-cycle": "warn",
+            "no-plusplus": "warn",
+            "no-continue": "warn",
+            "@typescript-eslint/no-dynamic-delete": "warn",
+            "no-return-assign": "warn",
+            "@typescript-eslint/prefer-nullish-coalescing": "warn",
+            "import/no-extraneous-dependencies": "warn",
+            "@typescript-eslint/no-unused-vars": "warn",
+        },
+    },
+    ...compat.extends('devextreme/typescript').map(config => {
+        const newConfig = {
+            ...config,
+            files: ["**/*.d.ts"],
+        };
+
+        if (config.rules) {
+          newConfig.rules = processDevExtremeRules(config.rules);
+        }
+
+        return newConfig;
+    }),
+    {
+        files: ["**/*.d.ts"],
+        plugins: {
+          i18n: i18N,
```

Comment:
You have already set this plugin in the global configuration - https://github.com/DevExpress/DevExtreme/pull/29110/files#diff-45126dc66c1f5318b0675e7af7a5cdfa492805e1824f11c7fe31a33169f6b712R43